### DISCUSSION TITLE
How to replace the navbar in the root layout for some pages?

### DISCUSSION THREAD

**Discussion by p1ckle-rick** [vercel/next.js#64811] [Help]
### Summary

I have a root layout that I want to use for most of the pages, including the root not-found and error pages.

However, the root layout has a navbar that I want to replace with another navbar for some pages.

When I try to conditionally render the navbar in the root layout and use `usePathname`, I get the following error.

```
You are attempting to export "metadata" from 
a component marked with "use client", which is disallowed. 
```

**Comment by pablojsx** [comment]
try hidden with tailwind?

**Reply by p1ckle-rick** [reply]
So you are suggesting to apply the hidden class on the navbar in the root layout?

But based on what condition?
